content/assignments/hello-world/_index.md

```diff
@@ -0,0 +1,151 @@
+         }
+
+         [TestMethod]
+         public void SecondTestToDemonstrateWhatATestMethodLooksLike()
+         {
+            //test code here
+         }
+      }
+   ```
+1. Run all of your tests using the Test Explorer window.
+   1. Mac Users: 
+      1. **View** > **Tests**  
+      1. Select the green triangle on the left of the Test Explorer. 
+      1. Step through the solution until you find the test that failed. 
+      1. [Guide](https://learn.microsoft.com/en-us/visualstudio/mac/testing?view=vsmac-2022#running-tests) to running tests in Visual Studio for Mac
+   1. Windows Users 
+      1. **Test** > **Windows** > **Test Explorer**
+      1. Select the `Run All` icon to run all of your tests. 
+      1. In the Test Explorer, step through your solution until you find the failed test.
+      1. [Guide](https://learn.microsoft.com/en-us/visualstudio/test/run-unit-tests-with-test-explorer?view=vs-2022) to running tests in Visual Studio.
+
+
+1. Visual Studio opens a Test Pane to display test results every time it runs a test. 
+You should see that one of the tests has failed and the other passed.  We have not touched any code yet. Use the Test Pane to navigate into which specific test has failed.
+
+   In this case, the `TestIfSayHelloReturnsCorrectValue()` test, expected our code to output `Hello, World!` Instead, it provided `&&&` causing it to fail.  This particular test has a message about why it failed.  `&&&` is not the correct message.
+
+   The `TestSayHelloReturnTypeIsString()` test passed.  This should pass because the `SayHello()` method is a string method.  You can verify this in line 8 of the `SayHelloClass`.  Use this test to help you explore the Test Pane and testing process. 
+
+1. Try running each test individually before moving on to the next step.  You can select individual tests by right-clicking on the test name either directly in the test class or the Test Explorer window.
+
+   You will still have the same results as before, but now you have the skills to run tests all together or one at a time.  This is something that may come in handy as you work on larger assignments that may have many tests.
+
+## Your Task
+
+Your task is simple: make the program print out the string `Hello, World!`. Edit the code in the `SayHelloClass` class, within the `SayHello()` method. Verify your code with the auto-grading tests. 
+
+  {{% notice orange "Warning" "rocket" %}}
+   The auto-grading tests are VERY exacting. A difference of a single character will result in a failed test. The tests are also case-sensitive. You’ll need to pay attention to detail to correctly complete your assignments.
+  {{% /notice %}}
+
+## Submitting Your Work
+
+Once you pass all of the tests, `add`, `commit` and `push` your code to GitHub.
+
+Visit your assignment repository page to see the results of your assignment.  If you passed, you'll see a green checkmark. If you see a red X, then your assignment is not yet correct.
```

Comment:
```suggestion
Visit your assignment repository page to see the results of your submission.  A green checkmark means the  submission has successfully passed the tests.  If you see a red X, then your submission is not yet correct.
```

Reply:
Suggest adding after the last sentence -> If you get stuck reach out to your TA or fellow students on slack.